services/notifications/setup.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+import re
+import sys
+from pathlib import Path
+
+from setuptools import find_packages, setup
+
+
+def read_reqs(reqs_path: Path) -> set[str]:
+    return {
+        r
+        for r in re.findall(
+            r"(^[^#\n-][\w\[,\]]+[-~>=<.\w]*)",
+            reqs_path.read_text(),
+            re.MULTILINE,
+        )
+        if isinstance(r, str)
+    }
+
+
+CURRENT_DIR = Path(sys.argv[0] if __name__ == "__main__" else __file__).resolve().parent
+
+NAME = "simcore-service-notifications"
+VERSION = (CURRENT_DIR / "VERSION").read_text().strip()
+AUTHORS = ("Andrei Neagu (GitHK)",)
+DESCRIPTION = "Service for executing commands on docker nodes"
```

Comment:
I don't think this description is accurate

Reply:
updated